### ISSUE TITLE
ENet is not a constructor

### ISSUE THREAD

**Issue by WesCrusher** [lawrencebensaid/homebridge-enet#2]
@lawrencebensaid 
Upon installing this plugin I get the following error:
TypeError: enet_api_1.ENet is not a constructor
at new ENetPlatform (/homebridge/node_modules/homebridge-enet/src/dynamic-platform.ts:60:19)
at /homebridge/node_modules/homebridge/src/server.ts:473:40
at Array.forEach ()
at Server.loadPlatforms (/homebridge/node_modules/homebridge/src/server.ts:400:27)
at Server.start (/homebridge/node_modules/homebridge/src/server.ts:176:29)

Can you help me out?

**Comment by fierland** [comment]
just installed, got same error

**Comment by shanse01** [comment]
Same here, addressed this at end of July'22, so far no response. I would be really happy to have this plugin working and would be willig to donate once it is working as this would save me some invest in new hardware.

**Comment by huglester** [comment]
@lawrencebensaid hey. Maybe you can tell how much we can pay so you can make it work please?
I could pay in crypto

**Comment by huglester** [comment]
@shanse01 I got same error

I am trying to debug a little:

```
this.log.info(enet_api_1.ENet)

gives:

{ default: [class ENet] }
```

so far I am not expert in typescript/javascript, would like to help here.. but maybe you got ideas
Maybe it is something to do with "export default.." etc.

**Comment by huglester** [comment]
I fixed this error using patch:
https://github.com/lawrencebensaid/enet-api/pull/3

but as mentioned in PR, next error occurs which, for now I don't understand how to fix

**Comment by lawrencebensaid** [comment]
`ENet is not a constructor` error should be fixed in v1.1.0.
